src/index-template.ejs

```diff
@@ -8,5 +8,7 @@
 </head>
 <body>
     <div id="app"><%= htmlWebpackPlugin.options.appcontent %></div>
+    <script type="text/javascript" src="http://scratchblocks.github.io/js/scratchblocks-v3.1-min.js"></script>
+    <script type="text/javascript" src="http://scratchblocks.github.io/js/translations-all-v3.1-min.js"></script>
```

Comment:
See the updated #70, which uses a new browser-build which includes the translations.